templates/automotive/_chassis_animation.html

```diff
@@ -0,0 +1,1613 @@
+        </g>
+        <g id="Group-2231">
+          <mask id="mask-26" fill="white">
+          <use xlink:href="#path-25"/>
+          </mask>
+          <path id="Fill-2229" fill="#fff" mask="url(#mask-26)" d="m8.0448 3.1968c-0.2381 0.0544-0.4745 0.2874-0.6089 0.6191-0.199 0.4933-0.0867 1.0954 0.2364 1.3064 0.1089 0.0901 0.2382 0.1156 0.3674 0.0867 0.2365-0.0561 0.4678-0.2926 0.5716-0.6464 0.199-0.4932 0.085-1.0954-0.2263-1.308-0.1054-0.068-0.2228-0.085-0.3402-0.0578m-1.7877 1.5615c-0.4796 0.1105-0.9049 0.3878-1.3029 0.762l0.4661 1.1328c0.2806-0.2874 0.5528-0.4694 0.8641-0.5409 1.0324-0.2364 1.8506 0.711 1.9203 2.099l0.99-0.2262c-0.0391-1.0665-0.3487-1.9527-0.8573-2.5072-0.1157 0.068-0.2262 0.119-0.3385 0.1445-0.165 0.0375-0.3283 0.0205-0.495-0.0731-0.2704-0.1633-0.4252-0.4712-0.466-0.8505-0.2382-0.0646-0.5103-0.0017-0.7808 0.0596m-1.5427 1.0409c-0.5852 0.7008-1.0257 1.6398-1.2264 2.7012 0.2398 0.1701 0.3538 0.546 0.3521 0.995-0.0017 0.4491-0.1174 0.8796-0.3572 1.1586 0.1956 0.971 0.6293 1.708 1.2111 2.143l0.5136-1.3c-0.5085-0.45-
```

Comment:
We also have `debounce` in three other places:
- https://github.com/canonical-web-and-design/ubuntu.com/blob/master/static/js/src/release-chart.js#L36
- https://github.com/canonical-web-and-design/ubuntu.com/blob/master/static/js/src/imageBuilder.js#L355
- https://github.com/canonical-web-and-design/ubuntu.com/blob/master/static/js/src/desktop-statistics.js#L3

Out of the scope of this PR but we should make that a utility function and import it where needed.